### PR TITLE
Update release note for 2.11.0

### PR DESCRIPTION
**Pull request by Technoboy-**
### Motivation
As released 2.11.0, add release note for it

### CODE REVIEW

File: release-notes/versioned/pulsar-2.11.0.md [hunk 1]
```diff
@@ -0,0 +1,643 @@
...
...massakam), [mattisonchao](https://github.com/mattisonchao), [maxsxu](https://github.com/maxsxu), [mendonk](mendonk), [merlimat](https://github.com/merlimat), [michaeljmarshall](https://github.com/michaeljmarshall), [misselvexu](https://github.com/misselvexu), [momo-jun](https://github.com/momo-jun), [MTwz](https://github.com/MTwz), [nahguam](https://github.com/nahguam),[nicoloboschi](https://github.com/nicoloboschi), [nodece](https://github.com/nodece), [poorbarcode](https://github.com/poorbarcode), [rdhabalia](https://github.com/rdhabalia), [RobertIndie](https://github.com/RobertIndie), [samredai](https://github.com/samredai), [Shawyeok](https://github.com/Shawyeok), [Sherlock113](https://github.com/Sherlock113), [shibd](https://github.com/shibd), [Shoothzj](https://github.com/Shoothzj), [SignorMercurio](https://github.com/SignorMercurio), [Technoboy-](https://github.com/Technoboy-), [tjiuming](https://github.com/tjiuming), [tisonkun](https://github.com/tisonkun), [urfreespace](https://github.com/urfreespace), [wangjialing218](https://github.com/wangjialing218), [wenbingshen](https://github.com/wenbingshen), [youzipi](https://github.com/youzipi), [zwOvO](https://github.com/zwOvO).
```
Comment (momo-jun) [inline release-notes/versioned/pulsar-2.11.0.md:643]:
```suggestion
> [0xflotus](https://github.com/0xflotus), [315157973](https://github.com/315157973), [704237006](https://github.com/704237006), [AlphaWang](https://github.com/AlphaWang), [alpreu](https://github.com/alpreu), [AlvaroStream](https://github.com/AlvaroStream), [AnonHxy](https://github.com/AnonHxy),[Anonymitaet](https://github.com/Anonymitaet), [asafm](https://github.com/asafm), [cbornet](https://github.com/cbornet), [codelipenghui](https://github.com/codelipenghui), [coderzc](https://github.com/coderzc), [D-2-Ed](https://github.com/D-2-Ed), [DaveDuggins](https://github.com/DaveDuggins), [Demogorgon314](https://github.com/Demogorgon314), [ericsyh](https://github.com/ericsyh), [fantapsody](https://github.com/fantapsody), [futeng](https://github.com/futeng), [hangc0276](https://github.com/hangc0276), [heesung-sn](https://github.com/heesung-sn), [hrsakai](https://github.com/hrsakai), [HQebupt](https://github.com/HQebupt), [horizonzy](https://github.com/horizonzy), [Huanli-Meng](https://github.com/Huanli-Meng), [gaozhangmin](https://github.com/gaozhangmin), [ikilobyte](https://github.com/ikilobyte), [Jason918](https://github.com/Jason918), [komalatammal](https://github.com/komalatammal), [labuladong](https://github.com/labuladong), [larshp](https://github.com/larshp), [lgxbslgx](https://github.com/lgxbslgx), [lhotari](https://github.com/lhotari), [liangyepianzhou](https://github.com/liangyepianzhou), [lin-zhao](https://github.com/lin-zhao), [liudezhi2098](https://github.com/liudezhi2098), [liuzhuang2017](https://github.com/liuzhuang2017), [Mans2singh](https://github.com/mans2singh), [massakam](https://github.com/massakam), [mattisonchao](https://github.com/mattisonchao), [maxsxu](https://github.com/maxsxu), [mendonk](mendonk), [merlimat](https://github.com/merlimat), [michaeljmarshall](https://github.com/michaeljmarshall), [misselvexu](https://github.com/misselvexu), [momo-jun](https://github.com/momo-jun), [MTwz](https://github.com/MTwz), [nahguam](https://github.com/nahguam),[nicoloboschi](https://github.com/nicoloboschi), [nodece](https://github.com/nodece), [poorbarcode](https://github.com/poorbarcode), [Pradhyumnakashyap](https://github.com/pradhyumnakashyap), [rdhabalia](https://github.com/rdhabalia), [RobertIndie](https://github.com/RobertIndie), [samredai](https://github.com/samredai), [Shawyeok](https://github.com/Shawyeok), [Sherlock113](https://github.com/Sherlock113), [shibd](https://github.com/shibd), [Shoothzj](https://github.com/Shoothzj), [SignorMercurio](https://github.com/SignorMercurio), [Technoboy-](https://github.com/Technoboy-), [tjiuming](https://github.com/tjiuming), [tisonkun](https://github.com/tisonkun), [urfreespace](https://github.com/urfreespace), [wangjialing218](https://github.com/wangjialing218), [wenbingshen](https://github.com/wenbingshen), [youzipi](https://github.com/youzipi), [zwOvO](https://github.com/zwOvO).
```

Adding a few more new names that contributed to the 2.11 docs.

File: release-notes/versioned/client-java-2.11.0.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+- [improve][Java] Returns immutable data set when using TableView [#14833](https://github.com/apache/pulsar/pull/14833)
+- [improve][Java] Avoid timer task run before previous subscribe complete [#14818](https://github.com/apache/pulsar/pull/14818)
+- [improve][Java] Optimize pause when creating sub consumers in multi-topic consumer [#14566](https://github.com/apache/pulsar/pull/14566)
+- [cleanup][Java] Reduce code duplication in admin client [#16377](https://github.com/apache/pulsar/pull/16377)
+- [cleanup][Java] Remove redundant check for chunked message TotalChunkMsgSize in ConsumerImpl [#16797](https://github.com/apache/pulsar/pull/16797)
+- [fix][Java] Fixed ack failure in ReaderImpl due to null messageId [#17728](https://github.com/apache/pulsar/pull/17728)
+- [fix][Java] Unwrap completion exception for Lookup Services [#17717](https://github.com/apache/pulsar/pull/17717)
+- [fix][Java] Fix scheduledExecutorProvider not shutdown [#17527](https://github.com/apache/pulsar/pull/17527)
+- [fix][Java] Fix the message present in incoming queue after go to DLQ [#17326](https://github.com/apache/pulsar/pull/17326)
```
Comment (Anonymitaet) [inline release-notes/versioned/client-java-2.11.0.md:26]:
```suggestion
- [fix][Java] Fix the message present in an incoming queue after going to DLQ [#17326](https://github.com/apache/pulsar/pull/17326)
```

File: release-notes/versioned/client-websocket-2.11.0.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+id: client-websocket-2.11.0
+title: Client Websocket 2.11.0
+sidebar_label: Client Websocket 2.11.0
+---
+
+- [feature][WebSocket] Support encryption in websocket proxy [#16234](https://github.com/apache/pulsar/pull/16234)
```
Comment (Anonymitaet) [inline release-notes/versioned/client-websocket-2.11.0.md:7]:
```suggestion
- [feature][WebSocket] Support encryption in WebSocket proxy [#16234](https://github.com/apache/pulsar/pull/16234)
```

File: release-notes/versioned/pulsar-2.11.0.md (hunk 1 above)
Comment (Technoboy-) [reply release-notes/versioned/pulsar-2.11.0.md]: updated

File: release-notes/versioned/client-java-2.11.0.md (hunk 2 above)
Comment (Technoboy-) [reply release-notes/versioned/client-java-2.11.0.md]: updated

File: release-notes/versioned/client-websocket-2.11.0.md (hunk 3 above)
Comment (Technoboy-) [reply release-notes/versioned/client-websocket-2.11.0.md]: updated

File: release-notes/versioned/client-cpp-2.11.0.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+---
+id: client-cpp-2.11.0
```
[the remark this answers is not in the document]
Comment (Technoboy-) [reply release-notes/versioned/client-cpp-2.11.0.md]: Removed, thanks

**Review by momo-jun — APPROVED** [review]

**Comment by momo-jun** [comment]
@Technoboy- I did a couple of fixes when previewing this page:)

**Comment by Technoboy-** [comment]
> @Technoboy- I did a couple of fixes when previewing this page:)

Thanks

**Pull request merged**